spec/requests/api/v1/posters_request_spec.rb

```diff
@@ -120,6 +120,24 @@
     expect(response).to be_successful
     result = JSON.parse(response.body, symbolize_names: true)
     expect(result[:meta][:count]).to eq(result[:data].count)
+
+    #SHOW
+    get "/api/v1/posters/#{@poster_1[:id]}"
+
+    expect(response).to be_successful
+    result = JSON.parse(response.body, symbolize_names: true)
+    expect(result[:meta][:count]).to eq(1)
```

Comment:
We couldnt get this to dynamically call the correct value, so we are putting in one as the expected return, because we know when we are showing, creating, or updating one poster  we know the count length will always be one